.github/workflows/gcp-services.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate GCP Services
+on:
+  schedule:
+    - cron: "60 * * * *"
```

Comment:
is it every hours? can we do weekly basis?

Reply:
This is actually wrong syntax 🤦 
I'll update it to run every Monday morning